src/main/java/module-info.java

```diff
@@ -0,0 +1,8 @@
+/**
+ * This module provides a Java API that allows building User Defined Function scripts for Exasol.
+ */
+module com.exasol {
+    exports com.exasol;
```

Comment:
Package is too generic. I have the feeling that this might cause problems. Better rename package. That's a breaking change but better do it now than later.
```suggestion
    exports com.exasol.udf;
```